### DISCUSSION TITLE
Welcome to Cleanarr Discussions!

### DISCUSSION THREAD

**Discussion by appiekap653** [Cleanarr/Cleanarr#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shakir87** [comment]
Hi, how can this be installed in Docker on Unraid? is it possible? This is exactly what I'm looking for. Great project, great work. Thanks

**Reply by appiekap653** [reply]
It is still a work in progress, but when it is ready it sure will be installable on the common ways media servers are run. Docker, Unraid, Windows, Linux, MacOs etc.
It is written in .net core to be multiplatform.

**Comment by zaylman** [comment]
Wanted to check in on this project. Doesn't look like there has been much activity in the last year. Would love to implement this on my server.

**Reply by shakir87** [reply]
Me too! I hope someone picks up development for this project soon.

**Comment by hippytyre** [comment]
This project seems like exactly what I'm looking for. I'm currently using plex-cleaner but it only seems to work for TV shows.
Is there any way I can install this to test? I'm running Debian on a pi4.
